api/client/client.go

```diff
@@ -45,11 +44,20 @@ type Remote interface {
 func NewServer(addr string) Remote {
 	addrs := strings.Split(addr, ",")
 
+	for i := range addrs {
```

Comment:
consider using the "net/url" package here instead of using raw string operations:
https://golang.org/pkg/net/url/#example_URL